nixos/tests/mediawiki.nix

```diff
@@ -54,4 +54,15 @@ in
       assert "MediaWiki has been installed" in page
     '';
   };
+
+  nohttpd = testLib.makeTest {
+    name = "mediawiki-nohttpd";
+    nodes.machine = {
+      services.mediawiki.webserver = "none";
+    };
+    testScript = ''
+      start_all()
+      machine.wait_for_unit("phpfpm-mediawiki.service")
```

Comment:
Would be good to put this behind a webserver to test if  we get a HTTP/200.

Reply:
I tested the fastcgi socket instead.